modules/gitops-release-notes-1-14-1.adoc

```diff
@@ -0,0 +1,53 @@
+
+{gitops-title} 1.14.1 is now available on {OCP} 4.13, 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-14.1_{context}"]
+== Errata updates
+
+[id="RHEA-XXXX:NNNN-gitops-1-14-1-security-update-advisory_{context}"]
+=== RHEA-XXXX:NNNN - {gitops-title} 1.14.1 security update advisory
+
+Issued: 2024-10-29
+
+The list of security fixes that are included in this release are documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHEA-XXXX:NNNN[RHEA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-14-1_{context}"]
+== Fixed issues
+
+* Before this update, resource limits were set on the operator container as a best practice, which resulted in functional issues on clusters containing a large number of `Secrets` and `ConfigMaps`. This issue arises due to the operator container requiring more memory than the permitted limits. With this update, the resource limits set on the manager container have been removed to minimize the impact on functionality. Efforts are focused to optimize memory consumption for future releases. link:https://issues.redhat.com/browse/GITOPS-5665[GITOPS-5665]
+
+* Before this update, Argo CD could not obtain the appropriate TLS certificate for Helm OCI registries when the URL included a path or port number. With this update, a fix is introduced in upstream Argo CD to correctly parse the URL and return a valid certificate. link:https://issues.redhat.com/browse/GITOPS-5081[GITOPS-5081]
+
+* Before this update, you could not log in to the Argo CD web console UI after a Red Hat OpenShift Service on AWS cluster, that includes a GitOps Operator and an Argo CD instance configured with Dex-based SSO, resumed from hibernation. The login screen would display an error indicating an invalid redirect URI in the Dex configuration. This update fixes the issue by ensuring that the correct Dex redirect URL is updated in the Argo CD configuration whenever the Argo CD server route is modified. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
+
+[id="breaking-change-1-14-1_{context}"]
+== Breaking change
+
+[id="additional-configurations-for-cluster-scoped-rollouts-instance-installation_{context}"]
+=== Additional configurations for cluster-scoped rollouts instance installation
+
+* When you upgrade to {gitops-title} v1.14, if you want to create the cluster-scoped rollouts installation outside the default installation namespace, `openshift-gitops`, you must host it in the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable of the `Subscription` resource. The {gitops-title} Operator does not support cluster-scoped rollouts installation if the namespace is not defined in the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable.
```

Comment:
> host it

I'm not 100% certain what that this means. Add the namespace to the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable?  
Maybe an example (is this right?)

```
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: openshift-gitops-operator
  namespace: openshift-operators
# ...
spec:
  config:
    env:apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: openshift-gitops-operator
  namespace: openshift-operators
# ...
spec:
  config:
    env:
    - name: CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES
      value: openshift-gitops, <list of namespaces of cluster-scoped Argo CD instances>
    - name: ARGOCD_CLUSTER_CONFIG_NAMESPACES
      value: openshift-gitops, <list of namespaces of cluster-scoped Argo CD instances>
```

Reply:
Hello @mburke5678, If an example clears things, I have one written in a different PR that relates to this RN entry for the feature. An example under this [section](https://83242--ocpdocs-pr.netlify.app/openshift-gitops/latest/argo_rollouts/using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources.html) is something that I am proposing.

Reply:
@Dhruv-Soni11 Perfect. Thanks!

Reply:
Fixed.